src/tests/Redux-store.test.js

```diff
@@ -0,0 +1,25 @@
+/*eslint-disable*/
+import store from '../redux/configureStore';
```

Comment:
- [ ] It would be better if we have all our written test cases passed and marked green. However, we have some failing test as we can see in the screenshot below. Let us investigate and fix it accordingly :man_technologist: 

![Screenshot from 2022-09-29 11-44-33](https://user-images.githubusercontent.com/46388113/193023414-cd25cb5e-1c32-4fe1-9e4f-86983e199524.png)